test/sequential/test-net-better-error-messages-port.js

```diff
@@ -3,7 +3,7 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT);
+const c = net.createConnection(common.PORT, common.localhostIPv4);
```

Comment:
```suggestion
const c = net.createConnection(common.PORT);
```
Shouldn't you instead change the assertion line 13:
```js
assert.strictEqual(e.address, common.hasIPv6 ? '::1' : '127.0.0.1');
```

Reply:
FWIW I think `assert.match` would make more sense for this test.